lib/Alchemy/Phrasea/Application.php

```diff
@@ -418,7 +416,7 @@ public function getUnlockAccountData()
      */
     public function requireCaptcha()
     {
-        if ($this['conf']->get(['registry', 'webservices', 'captcha-enabled'])) {
+        if ($this['conf']->get(['registry', 'webservices', 'captchas-enabled'])) {
```

Comment:
Why are you renaming an existing configuration key ? It will break configurations for existing installs